### PR TITLE
[rioki-glow] Adds new port glow 0.2.0

### PR DESCRIPTION
**Pull request by rioki**
**Describe the pull request**

- #### What does your PR fix?

  Adds the new port of glow, in version 0.2.0

- #### Which triplets are supported/not supported? Have you updated the [CI baseline]
  All, yes.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  To the best of my knowledge yes. 
  The name conflict was discussed here: https://github.com/microsoft/vcpkg/discussions/24676

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by rioki** [comment]
Sorry about adding to many commits into the PR; I rebased it to limit it to the glow ones.

**Comment by rioki** [comment]
Sorry about running you CI a bit hot... UWP was not on my radar.

File: ports/rioki-glow/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "rioki-glow",
+  "version-semver": "0.2.0",
```
Comment (JonLiu1993) [inline ports/rioki-glow/vcpkg.json:3]:
````
The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
````

File: ports/rioki-glow/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    REF v0.2.0
+    SHA512 743c41f4fb501d8a360ca48932b870263db4683b9b25dd34224cfa07de3cea2cb5a19f74eb8193a771ec4fb5e54aeac1bad04dd77599169d256f49de16fc6a72
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+)
+
+vcpkg_cmake_install()
+
```
Comment (JonLiu1993) [inline ports/rioki-glow/portfile.cmake:13]:
```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/glow")
```

File: ports/rioki-glow/vcpkg.json (hunk 1 above)
Comment (rioki) [reply ports/rioki-glow/vcpkg.json:3]: Sounds reasonable, did not know that it existed.

File: ports/rioki-glow/portfile.cmake (hunk 2 above)
Comment (rioki) [reply ports/rioki-glow/portfile.cmake]: Sounds reasonable, did not know that it existed.

**Comment by rioki** [comment]
Please hold while I figure out the CMake configure vodoo.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Tested the usage successfully in my local machine

**Review by BillyONeal — APPROVED** [review]
New port checklist:
* Review the code ✅
* Check the name against https://repology.org/ ✅ (Does not exist, but the name chosen is specific enough that it won't get confused)
* Check the name against Bing/Google. (If the first results aren't C++ related, try "name C++") ✅ first result
* Check the source code for optional `find_package`s ✅
```
find_package(GLEW REQUIRED)
find_package(glm CONFIG REQUIRED)
```
* Check that the versioning scheme and license match what upstream says ✅
* Check that the source code comes from the upstream project's authoritative source. ✅
* Check that the generated usage is accurate. ✅ Thanks @JonLiu1993
* Check that the reported license is accurate. ✅

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**